index.html

```diff
@@ -48,4 +48,181 @@ <h2 class="hex connect">LET’S CONNECT</h2>
                     <a href="#"><img src="images/socials/Vector2.svg" /></a>
                 </li>
             </ul>
-        </section>
\ No newline at end of file
+        </section>
+        <section class ="works">
```

Comment:
- [ ] Kindly use the appropriate background color for your work section  as required by the previous reviewer it will make your design stick more to the figma design :+1:  to fix that I suggest you to use this color :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/227706874-dde22620-f8d6-47c1-83a8-838ec9990abc.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/227706908-f4885412-cb7a-44ce-a3c2-6ad1a8d58607.png)